README.md

```diff
@@ -1,3 +1,273 @@
+
+### landscape.iaas
+```yaml
+iaas:
+  region: <major region>-<minor region>             # region that Gardener will use for seed clusters
+  zones:
+    - <major region>-<minor region>-<zone>          # Example: europe-west1-b
+    - <major region>-<minor region>-<zone>          # Example: europe-west1-c     
+    - <major region>-<minor region>-<zone>          # Example: europe-west1-d
+  credentials:                                      # credentials to get access to the seed cluster through service account
+```
+Contains the information where Gardener will create seed clusters. By default, the *initial* seed component will create a seed resource using your base cluster as seed cluster.
+
+| Field | Type | Description | Example |Iaas Provider Documentation |
+|:------|:--------|:--------|:--------|:--------|
+|`region`|IaaS provider specific|Region where Gardener will create seed clusters and shoot clusters. | `europe-west1`|[GCP (HowTo)](https://cloud.google.com/kubernetes-engine/docs/how-to/managing-clusters#viewing_your_clusters), [GCP (overview)](https://cloud.google.com/docs/geography-and-regions); [AWS (HowTo)](https://docs.aws.amazon.com/cli/latest/reference/eks/describe-cluster.html), [AWS (Overview)](https://docs.aws.amazon.com/general/latest/gr/rande.html)|
+|`zones`|IaaS provider specific|Zones where Gardener will create seed clusters and shoot clusters. |`europe-west1-b`|[GCP (HowTo)](https://cloud.google.com/kubernetes-engine/docs/how-to/managing-clusters#viewing_your_clusters), [GCP (overview)](https://cloud.google.com/docs/geography-and-regions); [AWS (HowTo)](https://docs.aws.amazon.com/cli/latest/reference/eks/describe-cluster.html), [AWS (Overview)](https://docs.aws.amazon.com/general/latest/gr/rande.html)|
+|`creds`|IaaS provider specific|Service account credentials in a provider-specific format. | `serviceaccount.json: { <credentials> }` (GCP) | [GCP](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys), [AWS](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html#id_users_service_accounts)|
+
+The service account credentials will be used to give Gardener access to your base cluster:
+* To create a secret that will be used on the Gardener dashboard to create shoot clusters.
+* The control plane of the seed clusters will use this secret to store the etcd backups of the shoot clusters.
+
+### landscape.etcd
+```yaml
+etcd:
+  backup:
+    type: <gcs|s3>                    # type of blob storage
+    region: (( iaas.region ))         # region of blob storage (default: same as above)
+    credentials: (( iaas.creds ))     # credentials for the blob storage's IaaS provider (default: same as above)
```

Comment:
Should this be "(( iaas.credentials ))"?

Reply:
Yes, thanks!

Reply:
done